### ISSUE TITLE
Bug report - stylis removes declaration with quotes

### ISSUE THREAD

**Issue by felixmosh** [thysultan/stylis.js#46]
```css
/* @scope  */
.foo{ font-family: "Goudy Bookletter 1911"}
```

it removes the font-family declaration entirely.

**Comment by thysultan** [comment]
Thanks, i think this is a bug with auto semicolon insertion, i.e

```css
.foo{font-family: "Goudy Bookletter 1911";}
``` 
works.

I will push a patch for this.

**Comment by thysultan** [comment]
Patched in [3.2.4](https://github.com/thysultan/stylis.js/blob/master/CHANGELOG.md#324-july-17-2017)
